src/Api/Managers/DocumentManager.cs

```diff
@@ -50,6 +50,9 @@ internal EntityResult UpdateDocument(int topicId, string identity, string htmlCo
                 DbContext.Add(document);
             }
 
+            // Angular app trims img-tags to non-valid xml, let us repair it
+            htmlContent = Regex.Replace(htmlContent, "<img ([^>]*)?>", "<img $1 />");
```

Comment:
Not sure if this is even allowed, but can we be 100% sure that there will be no `<img src="blabla"></img>` tags? Because if this apperas in the document, we have a closing img tag without an opening one